lib/api/core/storage/storageEngine.js

```diff
@@ -86,6 +87,8 @@ class StorageEngine {
     await Bluebird.all(promises);
 
     await this._populateIndexCache();
+    // @todo unit tests
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)